bagisto/bagisto
# Optimization #4680

## Description
Link: https://github.com/bagisto/bagisto/issues/4680

## For Developers
- In the `Cart` model, already `item` is eagerly loaded. So not needed lazy again.
- Hold the `product_attribute_values` in the `Product` model.
- Given the method to refresh the singleton property.
- Hold the loaded product in the cart facade.

## How To Test This?
- As this is optimization, need to check thoroghly.
- Please note debug bar also produces a load on the project.

## Suggestion
- On the cart page, we can load the cart items later by calling by ajax.
- Rare case: If still cart item is increased too much we can give a load more also.

## Results

- Before

    ![before](https://user-images.githubusercontent.com/68321766/126308303-5b8c4d31-2148-4032-b569-3c5b419d1197.png)

- After

    ![after](https://user-images.githubusercontent.com/68321766/126308327-c7386713-f388-4447-bd13-b3467ae80119.png)
